lib/trln_argon/controller_override/local_filter.rb

```diff
@@ -33,6 +35,15 @@ def filtered_results_total
           filtered_results_query_response['response']['numFound']
       end
 
+      def filters_applied?
```

Comment:
I think it would be better if this method accepted search_state as an argument rather than addressing it directly:

```
def filters_applied?(options={})
  search_state = options.fetch(:search_state, nil)
  result_count = options.fetch(:result_count, nil)
  ....
end
```

I think the name of the method is misleading. It's really checking to see whether the prompt to remove filters should be displayed. I'd consider naming it something like `def show_link_to_remove_filters?`